packages/gtmcore/gtmcore/configuration/configuration.py

```diff
@@ -33,6 +33,7 @@ class Configuration(object):
     """Class to interact with LabManager configuration files    
     """
     INSTALLED_LOCATION = "/etc/gigantum/labmanager.yaml"
+    USER_LOCATION = "/mnt/gigantum/.labmanager/labmanager.yaml"
```

Comment:
maybe use `config.yaml` to indicate to the user it's the config file?

```suggestion
    USER_LOCATION = "/mnt/gigantum/.labmanager/config.yaml"
```